examples/Bunch/include/simulation_defines/param/laserConfig.param

```diff
@@ -74,6 +74,10 @@ BOOST_CONSTEXPR_OR_CONST float_64 PULSE_INIT = 20.0;
 /* laser phase shift (no shift: 0.0) */
 BOOST_CONSTEXPR_OR_CONST float_X LASER_PHASE = 0.0; /* unit: rad, periodic in 2*pi */
 
+/* Use only the 0th Laguerremode for a standard Gaussian */
+BOOST_CONSTEXPR_OR_CONST uint32_t MODENUMBER = 0;
```

Comment:
please rebase against `dev` and use `constexpr` directly where `BOOST_CONSTEXPR_OR_CONST` is used :)